pages.es/common/zstdmt.md

```diff
@@ -0,0 +1,7 @@
+# zstdmt
+
+> Este comando es un alias de `zstd --threads 0` (que establece el número de hilos (working threads) al número de núcleos de CPU físicos).
+
+- Muestra la documentación del comando original:
```

Comment:
```suggestion
- Ver documentación para el comando original:
```

Reply:
I have unified the 7 PRs to let the user know what to do. to get documentation provided by tldr. Thank you :)